src/widgets/visit/edit-visit-component.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useEffect } from "react";
+import styles from "./edit-visit.css";
+import { useCurrentPatient } from "@openmrs/esm-api";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+import dayjs from "dayjs";
+import { DataCaptureComponentProps } from "../shared-utils";
+import { getPatientVisits } from "./visit-resource";
+import { getStartedVisit, visitMode, visitStatus } from "./visit-utils";
+
+export default function EditVisit(props: EditVisitProps) {
+  const [patientVisits, setPatientVisits] = useState<any[]>([]);
```

Comment:
Well the `Array<any>` here is essentially meaningless. The `[]` is already that. Can we be more specific?